src/psyclone/tests/psyir/backend/sympy_writer_test.py

```diff
@@ -49,6 +49,9 @@
 def test_sym_writer_boolean():
     '''Test that booleans are written in the way that SymPy accepts.
     '''
+    sympy_writer = SymPyWriter({})
+    lit = Literal("true", BOOLEAN_TYPE)
+    # Also test that not specifying a type map as argument works:
```

Comment:
I think you're missing a line that tests `sympy_writer(lit)` here?

Reply:
I think I actually have a duplicated line - the two required tests are just following in the next two lines:
```
    lit = Literal("true", BOOLEAN_TYPE)
    assert sympy_writer(lit) == "True"
    lit = Literal("false", BOOLEAN_TYPE)
    assert sympy_writer(lit) == "False"
````
I've removed the duplicated line.

Reply:
I can't see where you've done that (maybe due to a merge confusion?)

Reply:
I have no idea what had happened here, I am pretty certain that I removed the duplicated line. I have just redone this, making one separate test for the constructor, and removing the duplicated line. I hope it should be all ok now.

Reply:
Ah, I see. I'd misunderstood!